backend/src/main/java/ai/verta/modeldb/authservice/RoleServiceUtils.java

```diff
@@ -1150,6 +1153,150 @@ public WorkspaceDTO getWorkspaceDTOByWorkspaceId(
     }
   }
 
+  private VisibilityEnum.Visibility fromResourceVisibility(
+      Integer workspaceType, ResourceVisibility resourceVisibility) {
+    if (workspaceType == WorkspaceType.ORGANIZATION.getNumber()) {
+      switch (resourceVisibility) {
+        case PRIVATE:
+          return VisibilityEnum.Visibility.PRIVATE;
+        case ORG_DEFAULT:
+          return VisibilityEnum.Visibility.ORG_DEFAULT;
+        case ORG_SCOPED_PUBLIC:
+          return VisibilityEnum.Visibility.ORG_SCOPED_PUBLIC;
+        default:
+          return VisibilityEnum.Visibility.UNRECOGNIZED;
+      }
+    }
+    return VisibilityEnum.Visibility.PRIVATE;
+  }
+
+  @Override
+  public ResourceVisibility getProjectVisibility(
+      String projectId, String workspaceName, Integer workspaceType) {
+    ResourceType resourceType =
+        ResourceType.newBuilder()
+            .setModeldbServiceResourceType(ModelDBServiceResourceTypes.PROJECT)
+            .build();
+    Resources resources =
+        Resources.newBuilder()
+            .setResourceType(resourceType)
+            .setService(ServiceEnum.Service.MODELDB_SERVICE)
+            .addResourceIds(projectId)
+            .build();
+    List<GetResourcesResponseItem> responseItems =
+        getAllResourceItems(workspaceName, Optional.of(resources));
+    if (responseItems.size() > 1) {
+      LOGGER.warn(
+          "Role service returned "
+              + responseItems.size()
+              + " resource response items fetching project visibility, but only expected 1.");
+    }
+    final Optional<GetResourcesResponseItem> responseItem =
+        responseItems.stream()
+            .filter(
+                item ->
+                    item.getResourceType().getModeldbServiceResourceType()
+                            == ModelDBServiceResourceTypes.PROJECT
+                        && item.getService() == ServiceEnum.Service.MODELDB_SERVICE)
+            .findFirst();
+    if (responseItem.isPresent()) {
+      return responseItem.get().getVisibility();
+    }
+    return ResourceVisibility.PRIVATE;
+  }
+
+  private ResourceVisibility getResourceVisibility(
```

Comment:
I don't see this being used where. Could you clarify where it's used? I mainly want to understand using that visibility enum.

Reply:
It's not used, removing it.